test/run

```diff
@@ -6,7 +6,11 @@
 # IMAGE_NAME specifies a name of the candidate image used for testing.
 # The image has to be available before this script is executed.
 #
-declare -a WEB_APPS=({gunicorn-config-different-port,gunicorn-different-port,django-different-port,standalone,setup,setup-cfg,setup-requirements,django,numpy,app-home,npm-virtualenv-uwsgi,locale,mod-wsgi,pipenv,pipenv-and-micropipenv-should-fail,pin-pipenv-version,app-module{% if spec.version.startswith("3.") %},micropipenv,micropipenv-requirements{% endif %}}-test-app)
+{% if spec.minimal %}
+declare -a WEB_APPS=({gunicorn-config-different-port,gunicorn-different-port,django-different-port,standalone,setup,setup-requirements,django,numpy,app-home,locale,pipenv,pipenv-and-micropipenv-should-fail,pyuwsgi-pipenv,app-module,micropipenv}-test-app)
+{% else %}
+declare -a WEB_APPS=({gunicorn-config-different-port,gunicorn-different-port,django-different-port,standalone,setup,setup-cfg,setup-requirements,django,numpy,app-home,npm-virtualenv-uwsgi,locale,mod-wsgi,pipenv,pipenv-and-micropipenv-should-fail,pin-pipenv-version,app-module,pyuwsgi-pipenv{% if spec.version.startswith("3.") %},micropipenv,micropipenv-requirements{% endif %}}-test-app)
+{% endif %}
```

Comment:
For readability and easier maintainability, I think it'd be better to separate it into:

1. Tests for all images
2. Tests for full (non-minimal) images.

And then for full images just combine 1 & 2.

Reply:
Done. Hope it'll work in Bash.